spec/std/isa/ext/V.yaml

```
@@ -15,6 +15,212 @@ versions:
 description: |
   TODO
 params:
+  ELEN:
+    description: |
+      The maximum size in bits of a vector element that any operation can produce or consume, _ELEN_ {ge} 8, which
+      must be a power of 2.
+    schema:
+      type: integer
+      enum: [8, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096, 8192, 16384, 32768, 65536]
+  VLEN:
+    description: |
+      The number of bits in a single vector register, _VLEN_ {ge} ELEN, which must be a power of 2, and must be no greater than 2^16^.
+    schema:
+      type: integer
+      enum: [8, 16, 32, 64, 128, 256, 512, 1024, 2048, 4096, 8192, 16384, 32768, 65536]
+  SEWmin:
+    description: |
+      Implementations must provide fractional LMUL settings that allow the
+      narrowest supported type to occupy a fraction of a vector register
+      corresponding to the ratio of the narrowest supported type's width to
+      that of the largest supported type's width.  In general, the
+      requirement is to support LMUL {ge} SEW~MIN~/ELEN, where SEW~MIN~ is
+      the narrowest supported SEW value and ELEN is the widest supported SEW
+      value.
+    schema:
+      type: integer
+      enum: [8, 16, 32, 64]
+  VECTOR_AGNOSTIC_BEHAVIOR:
+    description:
+      When a set is marked agnostic, the corresponding set of destination
+      elements in any vector destination operand can either retain the value
+      they previously held, or are overwritten with 1s.
+    schema:
+      type: string
+      enum: ["never_overwritten", "custom"]
+  SUPPORT_FRACTIONAL_LMUL_BEYOND_REQUIRED:
+    description:
+      For a given supported fractional LMUL setting, implementations must support
+      SEW settings between SEW~MIN~ and LMUL * ELEN, inclusive.
+    schema:
+      type: string
+      enum: ["no_unrequied_supported", "custom"]
+  VILL_SET_ON_RESERVED_VTYPE:
+    description:
+      The use of `vtype` encodings with LMUL < SEW~MIN~/ELEN is
+      __reserved__, but implementations can set `vill` if they do not
+      support these configurations.
+    schema:
+      type: boolean
+  VILL_IMPLICIT_ENCODING:
+    description: |
+      A small implementation supporting ELEN=32 requires only seven bits of state in `vtype`: two bits for `ma` and `ta`, two bits for `vsew[1:0]` and three bits for `vlmul[2:0]`.  The illegal value represented by `vill` can be internally encoded using the illegal 64-bit combination in `vsew[1:0]` without requiring an additional storage
```

nit: missing a period at the end.
```suggestion
      A small implementation supporting ELEN=32 requires only seven bits of state in `vtype`: two bits for `ma` and `ta`, two bits for `vsew[1:0]` and three bits for `vlmul[2:0]`.  The illegal value represented by `vill` can be internally encoded using the illegal 64-bit combination in `vsew[1:0]` without requiring an additional storage.
```